Content.Goobstation.Server/StationReport/StationReportSystem.cs

```diff
@@ -18,18 +17,22 @@ public override void Initialize()
 
     private void OnRoundEndTextAppend(RoundEndTextAppendEvent args)
     {
+        var reportDefaultFormLoc = ((PaperComponent) _prototypeManager.Index("NanoRepStationReport").Components["paper"].Component).Content; // Omu: Don't send a report that hasn't been filled in
```

Comment:
```suggestion
        var reportDefaultFormLoc = ((PaperComponent) _prototypeManager.Index("NanoRepStationReport").Components["Paper"].Component).Content; // Omu: Don't send a report that hasn't been filled in
```
I guess it's case-sensitive?